### ISSUE TITLE
Allow people to enable / disable revisions per post type

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Revisions#19]
We should allow people to enable or disable revisions for specific post types

**Comment by Drumbell** [comment]
I love the idea. Does this include post categories? For example if I wanted revisions on "Places" but didn't want them on "Personal Stories" or "Announcements" ideally having a way to remove revisions for these categories.

**Comment by stevejburge** [comment]
@Drumbell Yes, that's feasible too, although I suspect full taxonomy support might be a 2nd step after post types

**Comment by stevejburge** [comment]
We may show the revisions UI in a few too many areas right now.

For example, this is the resuable block screen in Gutenberg:

<img width="1084" alt="Screen Shot 2019-10-10 at 5 54 02 PM" src="https://user-images.githubusercontent.com/3868028/66609458-19f35300-eb87-11e9-8cbb-ac3d35459844.png">

**Comment by Drumbell** [comment]
Have never seen that view before, but indeed Revisions should be applied to the objects/page types  they are intended for.
